mhlib/app.go

```diff
@@ -269,3 +293,81 @@ func (a *App) render(configFile string) (*string, *string, *[]byte, error) {
 
 	return &chart, &chartVersion, &overrides, nil
 }
+
+
+func selfRender(templateValuesStr string, enableGomplate bool) (string, error) {
+	/*
+	This function will accept an input string and run it through the
+	  templating engine as both the values dictionary as well as the
+	  template string.  It will repeat the process until the templating
+	  result stops changing.
+	*/
+
+	type Values map[string]interface{}
+
+	lastRender := templateValuesStr
+	for i := 0; i < 10; i++ {
+
+		// Unmarshal the file as a values dict
+		vals := Values{}
+		err := yaml.Unmarshal([]byte(templateValuesStr), &vals)
+		if err != nil { panic(err) }
```

Comment:
We may not want to panic in the lib for all of these, since it will cause any wrapping program to exit.  SHOULD: Follow error handling pattern of other lib functions.

https://golang.org/doc/effective_go.html#panic